database/migrations/2018_04_09_162941_add_columns_to_users_table.php

```diff
@@ -16,7 +16,8 @@ class AddColumnsToUsersTable extends Migration
     public function up()
     {
         Schema::table(User::name(), function (Blueprint $table) {
-            $table->string('first_name')->nullable()->after('password');
+            $table->string('google_id')->nullable()->after('password');
```

Comment:
If you're going to be adding other social providers, I suggest you have `social_provider` and `social_id` columns

Reply:
I think it's just google right now, and it's because of its calendars and events.